student/models.py

```diff
@@ -59,6 +65,7 @@ class year, major, friends, etc. An object is only created for the
     first_name = models.CharField(max_length=255, default='', null=True)
     last_name = models.CharField(max_length=255, default='', null=True)
     disabilities = models.NullBooleanField(null=True, default=False)
+    favorite_num = 10;
```

Comment:
set favorite_num to IntegerField (but I'll assume this was done as a patch if postgres was buggy)